.github/workflows/ci.yml

```diff
@@ -22,6 +22,9 @@ jobs:
     name: '${{matrix.platform}} / Node.js ${{ matrix.node }}'
     runs-on: ${{matrix.platform}}
 
+    continue-on-error: >-  # require nodejs-16 to succeed in branch protection
+      ${{ matrix.node != '16' && true || false }}
+
```

Comment:
This is not needed. All should be required.

Reply:
@zkochan ah, okay. I tried to mirror what you've already got in branch protection. But if you just forgot to add others, then yes, it makes sense to drop this.

Reply:
Fixed. Resolving this thread.